octohatrack/helpers.py

```diff
@@ -7,25 +7,29 @@
 from .generatehtml import generate_html
 from functools import wraps
 
-# Always run on start import
-cache_file = "cache_file.json"
-cache = {}
+print(sys.argv)
```

Comment:
Whoops, debugging still in place. I'll remove it when I get to a machine.
On Feb 3, 2016 1:14 PM, "E. Dunham" <notifications@github.com> wrote:

> In octohatrack/helpers.py
> <https://github.com/LABHR/octohatrack/pull/61#discussion_r51670065>:
>
> > @@ -7,25 +7,29 @@
> >  from .generatehtml import generate_html
> >  from functools import wraps
> >
> > -# Always run on start import
> > -cache_file = "cache_file.json"
> > -cache = {}
> > +print(sys.argv)
>
> I'll happily use this for my own debugging, but perhaps you'll want the
> print to eventually be optional?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LABHR/octohatrack/pull/61/files#r51670065>.
>